### DISCUSSION TITLE
Troubleshoot Fail: No NDI Source listed when trying to add a Source

### DISCUSSION THREAD

**Discussion by machine1111-code** [DistroAV/DistroAV#1162] [Q&A - Support]
Hello,

I use the latest macos 15.1.1 (24B91) and OBS 30.2.3, trying to install the latest NDI 6.1. There is no NDI Source listed when trying to add a Source. I noticed there is a troubleshooting guide, and it lists some options to try, and none of the files existed.  

MacOS: ~/Library/Application Support/obs-studio/plugins/obs-ndi.plugin
MacOS: /Library/Application Support/NewTek/NDI/usr/local/lib/libndi*

The entire NewTek folder and plugins folder do not exist.

<img width="182" alt="Screenshot 2024-12-09 at 9 50 12 PM" src="https://github.com/user-attachments/assets/9096861f-42e2-4f60-9c53-c6a5da8f3125">

Now, I have no idea how to proceed.

Please help.

Thank you in advance.

**Comment by BASE-1000** [comment]
I have the same problem. Doesn't anyone answer this problem

**Comment by Trouffman** [comment]
We have update the Wiki for the installations steps : https://github.com/DistroAV/DistroAV/wiki/1.-Installation

And troubleshooting at : https://github.com/DistroAV/DistroAV/wiki/2.-Troubleshooting

I can suggest you both follow the installation steps gain, and look at the "Support Request" steps in `Create new issue in DistroAV/DistroAV: Support Request (beta)`
